harness/determined/estimator/_estimator_trial.py

```diff
@@ -430,6 +431,12 @@ def __init__(
 
         self._init_model()
 
+    @classmethod
+    def supports_average_training_metrics(cls: Type["EstimatorTrialController"]) -> bool:
+        # Return True with a warning in order to pass tests.
+        warnings.warn("EstimatorTrialController does not support average_training_metrics = True")
+        return True
```

Comment:
can you just make the change in the `_check_if_trial_supports_configurations()` instead?

This pattern of "everything is a trial" is going away, so the value of that check is approaching zero.  Part of the Trainer API is to move training-loop settings like average_training_metrics out of the expconf and into the Trainer APIs, where individual framework APIs can handle it themselves.

Reply:
Oh, do you just want me to remove `_check_if_trial_supports_configurations` entirely? `experiment_config.average_training_metrics_enabled` is the only thing it checks.  I can also remove `supports_average_training_metrics` everywhere, if we do that

Reply:
Doing the above.